src/resolvers/projectResolver.ts

```diff
@@ -1047,6 +1055,7 @@ export class ProjectResolver {
           projectId,
         },
       )
+      .orderBy(field, direction)
```

Comment:
For some reason, typeorm not recognizing the column. 
Usually we interpolate the entity like this.
```javascript
.orderBy(`projectUpdate.${field}`, direction)
```

Reply:
Nice, good catch